nautobot/core/templates/graphene/graphiql.html

```diff
@@ -30,6 +30,9 @@
   <link rel="stylesheet"
         href="{% static 'flatpickr-4.6.9/themes/light.min.css' %}"
         onerror="window.location='{% url 'media_failure' %}?filename=flatpickr-4.6.9/themes/light.min.css'">
+  <link rel="stylesheet" id="dark-theme"
+        href="{% static 'css/dark.css' %}?v{{ settings.VERSION }}"
+        onerror="window.location='{% url 'media_failure' %}?filename=css/dark.css'" disabled="disabled" />
   <link rel="stylesheet"
         href="{% static 'css/base.css' %}?v{{ settings.VERSION }}"
         onerror="window.location='{% url 'media_failure' %}?filename=css/base.css'">
```

Comment:
Should replace this section with an `{% include 'inc/media.html' %}` similar to `nautobot/core/templates/base.html`.